### PR TITLE
Pairing down GCE from guide to needed changes

### PR DESCRIPTION
**Pull request by tmjd**
Addresses #715

### CODE REVIEW

File: master/getting-started/kubernetes/installation/gce.md [hunk 1]
```diff
@@ -2,50 +2,22 @@
 title: Deploying Calico and Kubernetes on GCE
 ---
 
-These instructions allow you to set up a Kubernetes cluster with Calico networking on GCE using the [Calico CNI plugin][cni-plugin]. This guide does not setup TLS between Kubernetes components or on the Kubernetes API.
+GCE provides a platform that can be used to
+[deploy a Kubernetes cluster][running-GCE-on-K8s] with minimal customization
+when following one of the many possibilities for deploying Kubernetes and
```
Comment (djosborne) [inline master/getting-started/kubernetes/installation/gce.md]: "many possibilities" doesn't seem right.

File: master/getting-started/kubernetes/installation/gce.md [hunk 2]
```diff
@@ -2,50 +2,22 @@
...
-And set a default zone
-
-```shell
-gcloud config set compute/zone us-central1-a
-```
-
-### 1.2 Setting up GCE networking
-
-GCE blocks traffic between hosts by default; run the following command to allow Calico traffic to flow between
+In addition to allowing all traffic between the hosts for the Kubernetes
```
Comment (djosborne) [inline master/getting-started/kubernetes/installation/gce.md]: This statement makes it sound like you've already explained the need to "allow all traffic between the hosts".

Furthermore, is that really a requirement? What are the bare minimum connections calico needs on the host? We don't necessarily have to figure this out and document it now but we should silently leave it out if that's the case.

File: master/getting-started/kubernetes/installation/gce.md (hunk 1 above)
Comment (djosborne) [inline master/getting-started/kubernetes/installation/gce.md]: Can you clean up this paragraph overall? It's a bit hard to follow

**Comment by tmjd** [comment]
@djosborne PTAL

**Comment by caseydavenport** [comment]
So, I think this might be turning into the equivalent of our AWS reference guide (see here: http://docs.projectcalico.org/v2.1/reference/public-cloud/aws)

Also, you might be interested in this WIP that I never finished: https://github.com/projectcalico/calico/pull/388

**Comment by tmjd** [comment]
I did have the thought to relocate it with the AWS reference guide too.  Why did that doc not get added @caseydavenport ?
@djosborne What do you think about moving it to the reference section since that seems to be closer to what it will be?

**Comment by caseydavenport** [comment]
> Why did that doc not get added @caseydavenport ?

Because I never found the time to finish it off :)

Feel free to steal as much from that other PR as you'd like.  I do sort of think that's where we want this doc to go.

File: master/getting-started/kubernetes/installation/gce.md [hunk 3]
```diff
@@ -2,49 +2,21 @@
 title: Deploying Calico and Kubernetes on GCE
 ---
 
-These instructions allow you to set up a Kubernetes cluster with Calico networking on GCE using the [Calico CNI plugin][cni-plugin]. This guide does not setup TLS between Kubernetes components or on the Kubernetes API.
+There are several possibilities for [deploying Kubernetes on GCE][running-GCE-on-K8s]
+and with minimal adjustments Calico can be [installed](.) too.
```
Comment (djosborne) [inline master/getting-started/kubernetes/installation/gce.md]: Is this link necessary? If so, there needs to be more info on the point of this link. It isn't clear why / when the user would need to click it.

File: master/getting-started/kubernetes/installation/gce.md [hunk 4]
```diff
@@ -2,49 +2,21 @@
 title: Deploying Calico and Kubernetes on GCE
 ---
 
-These instructions allow you to set up a Kubernetes cluster with Calico networking on GCE using the [Calico CNI plugin][cni-plugin]. This guide does not setup TLS between Kubernetes components or on the Kubernetes API.
+There are several possibilities for [deploying Kubernetes on GCE][running-GCE-on-K8s]
```
Comment (djosborne) [inline master/getting-started/kubernetes/installation/gce.md]: Possibilities is vague word. I think you mean "There are several tools available which can be used to easily create kubernetes clusters on Google cloud."

File: master/getting-started/kubernetes/installation/gce.md [hunk 5]
```diff
@@ -2,49 +2,21 @@
...
+and with minimal adjustments Calico can be [installed](.) too.
+The information on this page provides the customization needed.
```
Comment (djosborne) [inline master/getting-started/kubernetes/installation/gce.md]: "The information on this page describes Calico's additional requirements in order for it to function properly on Google Cloud."

Wording still sounds a little off in my suggestion but something along those lines

File: master/getting-started/kubernetes/installation/gce.md [hunk 6]
```diff
@@ -2,49 +2,21 @@
...
 
-## 1. Getting started with GCE
+### Setting up GCE networking
 
-These instructions describe how to set up two CoreOS Container Linux hosts on GCE.  For more general background, see
-[the CoreOS on GCE documentation][coreos-gce].
+Setting up Kubernetes will require [some GCE firewall rules][default-GCE-networking]
```
Comment (djosborne) [inline master/getting-started/kubernetes/installation/gce.md]: In addition to the standard GCE firewall rules required by kubernetes, Calico will need a firewall rule to allow IP-in-IP traffic between hosts.

File: master/getting-started/kubernetes/installation/gce.md (hunk 4 above)
Comment (tmjd) [inline master/getting-started/kubernetes/installation/gce.md]: It was purposefully vague, we don't really care how Kubernetes gets installed.  If you feel like this is better it works for me.

File: master/getting-started/kubernetes/installation/gce.md (hunk 3 above)
Comment (tmjd) [inline master/getting-started/kubernetes/installation/gce.md]: Yeah not really necessary, I just thought it was nice to provide a link to one of the many ways to install k8s. I can drop it if you don't think it is useful.

Comment (tmjd) [inline master/getting-started/kubernetes/installation/gce.md]: Maybe the link isn't good because we don't really want to 'endorse' one way over another.

**Comment by tmjd** [comment]
This was replaced by #739
